src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -604,14 +606,16 @@ def create(self, uid, display_name, email=None, max_buckets=None,
 
     @allow_empty_body
     def set(self, uid, display_name=None, email=None, max_buckets=None,
-            suspended=None, daemon_name=None):
+            system=None, suspended=None, daemon_name=None):
```

Comment:
i am assuming this is the edit functionality? can you actually edit a user and make it not system/system?